### PR TITLE
New Port: Sonic TT 16-Bit

### PR DESCRIPTION
**Pull request by JeodC**
# New Port for Sonic Triple Trouble 16-Bit

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

**Comment by JeodC** [comment]
Ready for review :)

File: ports/sonictt16b/port.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "version": 2,
+  "name": "sonictt16b.zip",
+  "items": [
+    "Sonic Triple Trouble - 16-Bit.sh",
+    "portname/"
```
Comment (Cebion) [inline ports/sonictt16b/port.json:6]: Portname/

File: ports/sonictt16b/port.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      "action",
+      "adventure",
+      "arcade",
+      "platformer"
+    ],
+    "image": "screenshot.png",
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": []
```
Comment (Cebion) [inline ports/sonictt16b/port.json:27]: Architecture missing

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

File: ports/sonictt16b/port.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      "action",
+      "adventure",
+      "arcade",
+      "platformer"
+    ],
+    "image": "screenshot.png",
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": ["aarch64"]
```
Comment (kloptops) [inline ports/sonictt16b/port.json:27]: This should be in the `"arch"` tag.
```suggestion
    "reqs": [],
    "arch": ["aarch64"]
```

**Review by kloptops — CHANGES_REQUESTED** [review]
Just a small change in the `port.json`. 😄

File: ports/sonictt16b/port.json (hunk 3 above)
Comment (kloptops) [reply ports/sonictt16b/port.json:27]: The requirements attribute isn't for filtering architecture.

**Review by kloptops — APPROVED** [review]

**Pull request merged**